xform-marc21-to-xml/src/test/java/edu/stanford/SqliteUtils.java

```diff
@@ -0,0 +1,49 @@
+package edu.stanford;
+
+import org.sqlite.SQLiteDataSource;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+
+/**
+ *
+ */
+public class SqliteUtils {
```

Comment:
Should this class name be changed to `SqliteTestUtils` to make it clear that they are only used in the context of tests (I know that it's already under the test folder and all...)